paraphraser/__init__.py

```diff
@@ -11,20 +11,23 @@ def paraphrase(data:str,lang:str,mode:str)->str:
     """
 
     url = "https://www.paraphraser.io/frontend/rewriteArticleBeta"
+    
+    session = requests.Session()
+    s = session.get('https://www.paraphraser.io')
 
     headers = {
-    "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/116.0",
+    "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/118.0",
     "Accept": "application/json, text/javascript, */*; q=0.01",
     "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
     "X-Requested-With": "XMLHttpRequest",
     "Origin": "https://www.paraphraser.io" ,
-    "Referer": "https://www.paraphraser.io/fr/outil-de-paraphrase"
+    "Referer": "https://www.paraphraser.io/"
     }
 
-    body = "data="+parser.quote_plus(data)+'+&mode='+mode+'&lang='+lang
+    body = "data="+parser.quote_plus(data)+'+&mode='+mode+'&lang='+lang+'&captcha='
 
     LOG.info(f"post request body: {body}")
-    response = requests.post(url,  headers=headers, data=body)
+    response = requests.post(url,  headers=headers, data=body, cookies=s.cookies.get_dict())
```

Comment:
You made a session `session` and didn't use it.
You can replace that line with `response = session.post(url,  headers=headers, data=body)` and the output would be the same (`requests.Session` object stores headers and cookies obtained inside of it)